eva-server/src/main/java/uk/ac/ebi/eva/server/ws/VariantWSServer.java

```diff
@@ -206,4 +206,17 @@ public QueryResponse checkVariantExists(@PathVariable("variantId") String varian
         }
     }
 
+    @RequestMapping(value = "/countVariants", method = RequestMethod.GET)
+    public QueryResponse countTotalNumberOfVariants() {
```

Comment:
Here I was referring to the method name, the endpoint is fine just as `count`. Sorry for the misunderstanding 😅

Reply:
Oh hahaha ok, hold on